samples/velocity/mark2.properties

```diff
@@ -6,7 +6,7 @@ mark2.regex.join=\\[connected player\\] (?P<username>[A-Za-z0-9_]{1,16}) \\(\\/(
 mark2.regex.quit=\\[connected player\\] (?P<username>[A-Za-z0-9_]{1,16}).* has disconnected(: )?(?P<reason>.*)
 plugin.monitor.crash-unknown-cmd-message=.*command.*does not exist.*
 # requires Velocity Utils plugin: https://forums.velocitypowered.com/t/velocity-utils-useful-commands-like-send-and-find/825
```

Comment:
Can you make this say `Alerts require the Velocity Utils plugin (not affiliated with mark2): <link>`